src/components/ui/filters/filtersData.js

```diff
@@ -0,0 +1,45 @@
+export let cityList = ['Vancouver', 'Washington', 'Barcelona', 'Manchester', 'Wien', 'Brussels']
```

Comment:
Const

Reply:
Can be dynamic:

`export const cityList = itemList.map(item => item.city)`

Reply:
ah, actually you would have duplicates